platform/barefoot/docker-syncd-bfn/Dockerfile.j2

```diff
@@ -20,7 +20,7 @@ RUN apt-get install -y \
         libcurl4 \
         libcurl4-gnutls-dev \
         libunwind8-dev \
-        libpython3.4 \
+        libpython3-dev \
```

Comment:
Why are these `*-dev` packages installed? What are they used for?

Reply:
They are needed for platform sdk, some unnecessary packages removed